c2corg_ui/templates/image/preview.html

```diff
@@ -0,0 +1,70 @@
+<%namespace file="helpers/detailed_images_attributes.html"
```

Comment:
Can't we use the exact file view.html instead of having preview.html and view.html ? It would be simpler to maintain, we could manage some special cases with `% if preview ....` as it is done for `version` for example.